ScoutSuite/providers/gcp/facade/facade.py

```diff
@@ -0,0 +1,12 @@
+
+class Facade:
```

Comment:
Is this an attempt to make an abstract interface for all cloud provider facades? Seeing how really different they are, I'm not sure we can really make them homogeneous...

Reply:
No it was simply for GCP, it avoids duplicating the `_get_client()` method, I can give it another name though

Reply:
For AWS, I called my base facade  `AWSBaseFacade`. I think it's less confusing than `Facade`.

Reply:
Renamed it to `GCPBaseFacade`!